container/single-instance-local/Dockerfile

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: MIT
+#!BuildTag: openqa-single-instance:latest opensuse/openqa-single-instance:latest opensuse/openqa-single-instance:%PKG_VERSION% opensuse/openqa-single-instance:%PKG_VERSION%.%RELEASE%
+
+# hadolint ignore=DL3006
+FROM opensuse/tumbleweed
+
+# labelprefix=org.opensuse.openqa-single-instance
+LABEL org.opencontainers.image.title="openQA single-instance container"
+LABEL org.opencontainers.image.description="A complete openQA instance composed of all necessary components to execute openQA tests including an openQA worker"
+LABEL org.opencontainers.image.version="%PKG_VERSION%.%RELEASE%"
+LABEL org.opensuse.reference="registry.opensuse.org/opensuse/openqa-single-instance:%PKG_VERSION%.%RELEASE%"
+LABEL org.openbuildservice.disturl="%DISTURL%"
+LABEL org.opencontainers.image.created="%BUILDTIME%"
+# endlabelprefix
+
+# installing more of packages that are selected in openqa-bootstrap.  Combining here saves installation time
+# hadolint ignore=DL3037
+RUN zypper in -y retry git make gcc perl \
+    openQA-bootstrap openQA-devel os-autoinst-distri-opensuse-deps \
+    qemu-arm qemu-ppc qemu-x86 qemu-tools sudo iputils\
+    qemu-hw-display-virtio-gpu qemu-hw-display-virtio-gpu-pci
```

Comment:
I'm wondering whether this list of packages is copied from somewhere and now duplicated.

Reply:
Well, mostly it is and I consider this bad as it makes the container file less maintainable.

Reply:
Fixed thanks!